Update docs.md

kayuapi opened the pull request:
Typo to the class Pastry.

kayuapi committed:
Update docs.md

Typo to the class Pastry.

amosfolz commented:
This is actually already correct. Why are you suggesting this change is needed?

See here https://github.com/userfrosting/pastries/blob/master/src/Database/Models/Pastries.php

lcharette commented:
Actually, I believe the model should be named `Pastries`. That whole tutorial probably need updating anyway.

kayuapi committed:
Update docs.md

kayuapi commented:
> This is actually already correct. Why are you suggesting this change is needed?
> 
> See here https://github.com/userfrosting/pastries/blob/master/src/Database/Models/Pastries.php

My bad, it's mistake on my side, I reverted the mistake and updated another typo.
